packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -285,23 +274,22 @@ const makeUserAdmin = (id: string) => {
 const makeUserAdminMutation = async (id: string | null) => {
   if (!id) {
     confirmUserToAdmin.value = false;
-    toast.error(`${t('state.admin_failure')}`);
+    toast.error(t('state.admin_failure'));
     return;
   }
   const variables = { uid: id };
-  await userToAdmin.executeMutation(variables).then((result) => {
-    if (result.error) {
-      toast.error(`${t('state.admin_failure')}`);
-    } else {
-      toast.success(`${t('state.admin_success')}`);
-      usersList.value = usersList.value.map((user) => {
-        if (user.uid === id) {
-          user.isAdmin = true;
-        }
-        return user;
-      });
-    }
-  });
+  const result = await userToAdmin.executeMutation(variables);
+  if (result.error) {
+    toast.error(t('state.admin_failure'));
+  } else {
+    toast.success(t('state.admin_success'));
+    usersList.value = usersList.value.map((user) => {
+      if (user.uid === id) {
+        user.isAdmin = true;
+      }
+      return user;
+    });
```

Comment:
Let's return the updated straightaway instead of mutating, [Ref](https://github.com/hoppscotch/hoppscotch/pull/3702/files#r1456059303).